tools/make/tidy.toml

```diff
@@ -175,7 +175,7 @@ exit_on_error true
 
 glob_pattern = set "./**/Cargo.toml"
 license = canonicalize LICENSE
-skip_path_prefixes = array "ffi/gn/vendor/" "ffi/gn/third_party_tools/"
+skip_path_prefixes = array "ffi/gn/vendor/" "ffi/gn/third_party_tools/" "ffi/diplomat/c/examples" "docs/tutorials/cargo_tests/"
```

Comment:
remove `ffi/diplomat/c/examples/fixeddecimal_tiny/icu_capi_staticlib_tiny/LICENSE`

Reply:
Actually everything in `ffi/diplomat/c/examples` is already being excluded so I went ahead and deleted LICENSE